lib/traject/macros/general.rb

```diff
@@ -5,7 +5,7 @@ module General
     def accumulate(&lambda)
       lambda do |record, accumulator, context|
         Array(lambda.call(record, context)).each do |v|
```

Comment:
@cbeer are you still considering renaming this lambda to distinguish it from `lambda do`?

Reply:
Yes, I'll rename all the lambda arguments for clarity.